### PR TITLE
mesheryctl mesh tests

### PR DESCRIPTION
**Pull request by Philip-21**
**Notes for Reviewers**

This PR fixes #

Mesheryctl unit tests for mesh pacakage

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit f847007355b2c45b0210557b0ffba1f095fb3bcc at: https://652555d8ce4de52f8de379ef--meshery-docs-preview.netlify.app

**Comment by saurabh100ni** [comment]
Thanks for raising the PR, let's Discuss this in the Meshery Development Call today,
Please add this as an agenda in the Doc  : [Doc](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit)

**Comment by MUzairS15** [comment]
// @zakisk @alphaX86

File: docs/pages/reference/mesheryctl/test.md
Comment (alphaX86) [inline docs/pages/reference/mesheryctl/test.md:1]: IDK what caused here but we need to remove these files after unit test gets over....

File: mesheryctl/doc/cmds.yml
Comment (alphaX86) [inline mesheryctl/doc/cmds.yml:1]: Same here...

File: mesheryctl/internal/cli/root/mesh/fixtures/token.golden [hunk 1]
```diff
@@ -0,0 +1 @@
...SmpSMncxWkZoT2IyTXliSFZhTWtaNVlWaHNhRHBhTW13d1lVaFdhU0o5Lk90aDJwYkJFNmFBcnBfUFVwR3E3b2ZsaEVWYmdsdTAtamdXNG44eWxHeVVTandOc0k4SmdoallIVGU5YjlUSzhWQUhoNVRyT0YwV1VRb0h4QVJGUmN6OHl2ZEdpbm1HcUZEZTd6RVpoSjZHZmNlZFl6bmpCc3FvVWthMTNXYzhvM0J2bGR2T2gtTjFGNzdHM3ZLenI0UEJaM2pXRHVEeWpjSUJnOTJVUzd0Nlg5Ymd6YklrT3lOOVhpWGVVNXQtbEJIamt2cklRazhqdWRKaTliOHVGaVBuMmdIMDVJbnhUdFJtSlFJdUhvSzV2WmxFQW0xN1J6ZER4WVI0cndqeTBqanFWdXdvWnBjbUJQM1dUNjdIVHhkYmo5N3hZM2IzNHh5ZFkxeVFVS09XR1NOckZVeXhMbW9QMmJUM24tQ0dVczJ1SWhnZExXNlZlNVQ1LV9tSGY0Z212X0NGWlFNelRsbjRFVmw2bTUxdjFxNXJzQmdfWmFuVmtXdGNHWF9ZSGs3WHpKdndXRDhvSmt5NzBleGUwYXJ3cmg2bjJkLU9jMi1Jc1F2OTBFM1hYeHBJcWxrckNfU3NiM1NpOU1jM1ptal9HY2JtOHVHbUZEejhaZEYxUEdpeDdKTjM3TzJyQnpaVldRaHFrZTV6MW42VUVITXJGSGJBNXBKVkxzUmE0ZUNBaFdwODVlZVV3ZjlUMnByc3FzNHBaMkh0eVpSMlBTdGFLZVFFai1SUXdvRHpDTEN4Zm85RnBvbEN6WmN3ZzRvLXhrb0Q0aS1MczIzODd0dm5xSTVESl8xaUlMX1hNTHByZXJtcDdxeGV2NEVDOW9abzdWenZmTDd4cDZTcnhIaldZQVpuZS12eURjQlhNZUlSMVVoeVdVZDQtaWJfZmxzdFVEME5XVV9ZIiwidG9rZW5fdHlwZSI6ImJlYXJlciIsInJlZnJlc2hfdG9rZW4iOiJXS3pZWW5BQkVJQkduekNfaWR2VW1IZUtsZlgzLWxjWm12TzBxY2ZCNlRzLm5kNXhXUFFIeWVTcTY0OUV2dy1tX2t3WDdqYWF1RDZiSExXTW9fQVhxZVUiLCJleHBpcnkiOiIyMDIxLTA2LTA0VDE3OjU5OjAzLjg0ODAyODAwOVoifQ"}
```
Comment (alphaX86) [inline mesheryctl/internal/cli/root/mesh/fixtures/token.golden:1]: Is this a real hardcoded token? 👀

File: mesheryctl/internal/cli/root/mesh/testdata/deploy.nginx.output.golden
Comment (alphaX86) [inline mesheryctl/internal/cli/root/mesh/testdata/deploy.nginx.output.golden:1]: Checking failed results it's great! But we might also need test cases if mesh cmd works IMO

File: docs/pages/reference/mesheryctl/test.md
Comment (Philip-21) [reply docs/pages/reference/mesheryctl/test.md:1]: yh i generated them locally i just wanted to see if te coverage will increase up to some point, i'll take them off

File: mesheryctl/internal/cli/root/mesh/fixtures/token.golden (hunk 1 above)
Comment (Philip-21) [reply mesheryctl/internal/cli/root/mesh/fixtures/token.golden:1]: yes, the token is used to run unit tests  for mesheryctl, the context env uses the token as a means to perform various tasks for each test

Comment (alphaX86) [reply mesheryctl/internal/cli/root/mesh/fixtures/token.golden:1]: Noted 👍

**Pull request merged**